<launch>
  <arg name="use_external_renderer" default="0" />
  <arg name="ignore_collisions" default="false" />
  <arg name="render_stereo" default="0" />

  <group ns="/control_nodes">
    <!-- Start autonomous nodes -->
    <node name="autonomous_control" pkg="autonomous_control" type="autonomous.py">
      <rosparam file="$(find universal_teleop)/launch/example_input_map.yml"/>

      <remap from="output/rateThrust" to="/uav/input/rateThrust"/>
      <remap from="output/reset" to="/uav/input/reset"/>
    </node>
  </group>

  <group ns="/image_processing_nodes">
      <!-- Start image processing nodes-->
    <node name="pathplanning" pkg="pathplanning" type="imageProcessing.py">

    </node>
  </group>

  <!--include core flightgoggles launch file-->
  <include file="$(find flightgoggles)/launch/core.launch">
    <arg name="use_external_renderer" value="$(arg use_external_renderer)"/>
    <arg name="render_stereo" value="$(arg render_stereo)"/>
    <arg name="ignore_collisions" value="$(arg ignore_collisions)"/>
  </include>

  <param name="/use_sim_time" value="false" />

</launch>
